tests/conftest.py

```diff
@@ -4,10 +4,10 @@
 import utils as utl
 # import snappi_convergence
 
-
-def pytest_exception_interact(node, call, report):
-    # end all pytests on first exception that is encountered
-    pytest.exit(call.excinfo.traceback[0])
+# TODO: uncomment this if we want pipeline to fail instantly 
```

Comment:
Please ensure we want to keep this commented code or remove it all together ?

Reply:
IMO we can remove it. There will be CL in PR, so that we can incorporate if required in future. @desaimg1 @Vibaswan